cmd/goal/clerk.go

```diff
@@ -566,6 +571,17 @@ func getProto(versArg string) config.ConsensusParams {
 	cvers := protocol.ConsensusCurrentVersion
 	if versArg != "" {
 		cvers = protocol.ConsensusVersion(versArg)
+	} else {
+		dataDir := maybeSingleDataDir()
+		if dataDir != "" {
+			client := ensureAlgodClient(dataDir)
+			params, err := client.SuggestedParams()
```

Comment:
I think that you don't want this one for a dryrun

Reply:
I think it doesn't hurt to have it. It will try to contact an algod but fall back to defaults if an algod can't be found.

Reply:
My concern is not that we would try and fail. My concern is that connecting to a live node is opposed to "dryrun" mode. Do you think we could skip this portion for when we specify dryrun ?

Reply:
If you have $ALGORAND_DATA defined and this quietly checks with your network for the current version, that sounds like a good thing to me. `--proto` preempts and overrides that so the check with algod is never done. I think this is a good way to guess a reasonable default.